packages/core/src/components/forms/_input-group.scss

```diff
@@ -162,6 +165,35 @@ $input-button-height-large: $pt-button-height !default;
     }
   }
 
+  &.#{$ns}-small {
+    .#{$ns}-button {
+      @include pt-button-height($pt-button-height-smaller);
+      margin: ($pt-input-height-small - $pt-button-height-smaller) / 2;
+    }
+
+    .#{$ns}-tag {
+      @include pt-button-height($pt-button-height-smaller);
+      margin: ($pt-input-height-small - $pt-button-height-smaller) / 2;
+    }
+
+    > .#{$ns}-icon,
+    .#{$ns}-input-action > .#{$ns}-spinner {
+      margin: ($pt-input-height-small - $pt-icon-size-standard) / 2;
+    }
+
+    .#{$ns}-input {
+      @include pt-input-small();
+
+      &:not(:first-child) {
+        padding-left: 25px;
+      }
+
+      &:not(:last-child) {
+        padding-right: 25px;
```

Comment:
please use variables to compute these values. see the other modifiers, like large.

Reply:
@giladgray can I create new variables for these? I wasn't able to find one that was exactly 25px and I thought 25px looked best.

Reply:
or is something like this acceptable? 
```
      &:not(:first-child) {
        padding-left: $pt-grid-size * 2.5;
      }
```